test/connectors/UpscanConnectorTest.scala

```diff
@@ -26,29 +26,29 @@ import org.mockito.ArgumentMatchers.{eq => eqTo, _}
 import org.mockito.Mockito._
 import org.scalatestplus.mockito.MockitoSugar
 import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import play.api.http.Status
 import play.api.libs.ws.ahc.AhcWSResponse
 import play.api.libs.ws.ahc.cache.CacheableHttpResponseStatus
 import play.api.libs.ws.{WSClient, WSRequest}
 import play.api.mvc.MultipartFormData
-import play.shaded.ahc.org.asynchttpclient.uri.Uri
-
-import scala.concurrent.Future
 import play.shaded.ahc.org.asynchttpclient.Response
+import play.shaded.ahc.org.asynchttpclient.uri.Uri
 
-class UpscanConnectorTest extends PlaySpec with MockitoSugar with SpecBase with SfusMetricsMock {
+class UpscanConnectorTest extends PlaySpec with GuiceOneAppPerSuite with MockitoSugar with SpecBase with SfusMetricsMock {
 
   val uploadUrl = "http://localhost/theUploadUrl"
-  val as = ActorSystem("test-system")
-  implicit val materializer = ActorMaterializer()(as)
   val contactDetails = ContactDetails("a", "b", "c")
 
   "Upload" should {
 
     "POST to Upscan" in {
       val wsClient = mock[WSClient]
       val wsRequest = mock[WSRequest]
+
+      implicit val materializer = app.materializer
```

Comment:
May I ask, why the previous materializer was not right?

Reply:
`ActorMaterializer()` is deprecated.